tests/snmpsim/ceraos.snmprec

```diff
@@ -1,2 +1 @@
-1.3.6.1.2.1.1.1.0|4|High capacity packet radio outdoor unit
-1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2281.1.20.2.2
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2281
```

Comment:
It's not no, it's used for the test units which should be as complete as possible. For instance someone could add an OS discovery based on sysName and match this device type before it got to this OS discovery. Having both sysObjectId and sysName in the test units means we catch more.